### ISSUE TITLE
C# file don't show any tree, even if click the refresh button

### ISSUE THREAD

**Issue by Meli55a** [patrys/vscode-code-outline#58]
First of all, thank you for your work!

- VS Code Version: 1.17.2 x64
- OS Version：Windows 10 x64

- ![image1](https://raw.githubusercontent.com/Meli55a/MarkdownPhotos/master/Issuse/20171107161405vscode-code-outline.png)

- Part of the extension associated with c # has been installed: ![image2](https://raw.githubusercontent.com/Meli55a/MarkdownPhotos/master/Issuse/20171107170223vscode-code-outline.png)

**Comment by wanderer06** [comment]
Do you see anything if you do `CTRL+SHIT+O` or _Go to Symbol in File_ command?
As mentioned before, this extension is based on the Symbols API.

**Comment by Meli55a** [comment]
I didn't see anything when i do `CTRL+SHIT+O`,  vscode prompt "no symbol information for this file", so , it doesn't support C#?

**Comment by wanderer06** [comment]
This extension does not do any language parsing itself, for obvious reason - it will need to have an implementation for every language. Instead it relies on the language server to provide symbols, and then just show that in a tree.
If you can get your Vvsode to show Symbols for C#, then this extension will undoubtedly work.

Have you tried this link?
https://code.visualstudio.com/docs/languages/csharp

**Comment by Meli55a** [comment]
Thank you. I've found the problem, when I open a `.cs` file directly, there is no tree display, but, when I open with a solution, `code-outline` can display the tree normally, so when I open the project with the solution, the vscode identifiles the symbol in the file.

My English is very poor, use translation software to communicate, excuse me...

Have a nice day!~haha

**Comment by wanderer06** [comment]
That's all right, I'm glad you figured it out!
Happy coding.
